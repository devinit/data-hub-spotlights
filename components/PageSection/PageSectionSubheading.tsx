import React, { FunctionComponent } from 'react';

const PageSectionSubHeading: FunctionComponent = ({ children }) => {
  return <h3 className="section__sub-heading">{children}</h3>;
};

export { PageSectionSubHeading };
