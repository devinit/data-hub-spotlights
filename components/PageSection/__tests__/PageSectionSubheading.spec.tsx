/**
 * @jest-environment jsdom
 */
import '@testing-library/jest-dom/extend-expect';
import React from 'react';
import * as TestRenderer from 'react-test-renderer';
import { PageSectionSubheading } from '../PageSectionSubheading';

describe('PageSectionSubheading', () => {
  test('renders correctly', () => {
    const renderer = TestRenderer.create(<PageSectionSubheading />).toJSON();

    expect(renderer).toMatchSnapshot();
  });

  test('It renders children correctly', () => {
    const renderer = TestRenderer.create(<PageSectionSubheading>{'My Child'}</PageSectionSubheading>);

    expect(renderer).toMatchSnapshot();
  });
});
