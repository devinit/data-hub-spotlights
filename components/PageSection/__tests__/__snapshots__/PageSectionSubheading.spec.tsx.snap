// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageSectionSubheading It renders children correctly 1`] = `
<h3
  className="section__sub-heading"
>
  My Child
</h3>
`;

exports[`PageSectionSubheading renders correctly 1`] = `
<h3
  className="section__sub-heading"
/>
`;
